state-chain/custom-rpc/src/lib.rs

```diff
@@ -1212,6 +1223,30 @@ where
 		)
 	}
 
+	fn cf_subscribe_scheduled_swaps(
```

Comment:
doc comment would be helpful here, in particular explaining that the asset order doesn't matter here, it'll get swaps in both directions for that pair